### DISCUSSION TITLE
2023/12/15/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BB%98%E5%9B%BE%E2%80%94%E2%80%94%E5%A4%8D%E5%8F%98%E5%87%BD%E6%95%B0%E5%92%8C%E6%9B%BC%E5%BE%B7%E5%8B%83%E7%BD%97%E9%9B%86/

### DISCUSSION THREAD

**Discussion by giscus** [ShwStone/shwblog#65] [Announcements]
# 2023/12/15/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BB%98%E5%9B%BE%E2%80%94%E2%80%94%E5%A4%8D%E5%8F%98%E5%87%BD%E6%95%B0%E5%92%8C%E6%9B%BC%E5%BE%B7%E5%8B%83%E7%BD%97%E9%9B%86/

https://shwst.one/2023/12/15/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BB%98%E5%9B%BE%E2%80%94%E2%80%94%E5%A4%8D%E5%8F%98%E5%87%BD%E6%95%B0%E5%92%8C%E6%9B%BC%E5%BE%B7%E5%8B%83%E7%BD%97%E9%9B%86/

**Comment by Panda-Ghost** [comment]
woo~~
